kubernetes/values.yaml

```diff
@@ -13,6 +13,11 @@ autoscaling:
   maxReplicas: 16
   targetCPUUtilizationPercentage: 50
 
+service:
+  type: clusterIP
+  port: 3000
+  # loadBalancerIP:
```

Comment:
Could you please remove the space here? Sorry! :see_no_evil:

Reply:
The space after the "#"? Sure